### PR TITLE
SCP-2151 add test coverage for flink artifact cmd

### PR DESCRIPTION
**Pull request by NicoleYWDu**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [ ] Leave this box unchecked if features are not yet available in production

What
----
related PR: https://github.com/confluentinc/cli/pull/2583

References
----------

Test & Review
-------------

### CODE REVIEW

File: test/fixtures/output/flink/artifact/create.golden [hunk 1]
```diff
@@ -0,0 +1,6 @@
++----------------+------------------+
+| Name           | my-custom-plugin |
+| Plugin ID      | ccp-123456       |
+| Version ID     |                  |
+| Content Format |                  |
```
Comment (brianstrauch) [inline test/fixtures/output/flink/artifact/create.golden:5]: Why are these empty? What are the default values when you first create a flink artifact?

File: test/flink_test.go [hunk 2]
```diff
@@ -1,8 +1,13 @@
 package test
 
-func (s *CLITestSuite) TestFlinkArtifactList() {
+func (s *CLITestSuite) TestFlinkArtifact() {
 	tests := []CLITest{
+		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar", fixture: "flink/artifact/create.golden"},
+		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar --description cliPluginTest", fixture: "flink/artifact/create.golden"},
+		{args: "flink artifact describe ccp-123456", fixture: "flink/artifact/describe.golden"},
 		{args: "flink artifact list", fixture: "flink/artifact/list.golden"},
+		{args: "flink artifact delete ccp-123456 --force ", fixture: "flink/artifact/delete.golden"},
```
Comment (brianstrauch) [inline test/flink_test.go:9]:
```suggestion
		{args: "flink artifact delete ccp-123456 --force", fixture: "flink/artifact/delete.golden"},
```

File: test/flink_test.go [hunk 3]
```diff
@@ -32,9 +33,14 @@ type flinkShellTest struct {
 	goldenFile string
 }
 
-func (s *CLITestSuite) TestFlinkArtifactList() {
+func (s *CLITestSuite) TestFlinkArtifact() {
 	tests := []CLITest{
+		{args: "flink artifact create my-flink-artifact --artifact-file test/fixtures/input/flink/java-udf-examples-3.0.jar", fixture: "flink/artifact/create.golden"},
```
Comment (brianstrauch) [inline test/flink_test.go:38]: I think this test should still pass if the ".jar" file is an empty file? (There's no reason for it to be 10KB as far as I can tell)

Comment (NicoleYWDu) [reply test/flink_test.go:38]: Yes, we use this for this system test repo too. https://github.com/confluentinc/cc-system-tests-scp-flink-udf/tree/master/resources

**Comment by NicoleYWDu** [comment]
@brianstrauch looked into ci failure. `test/flink_test.go:3: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/confluentinc/) -s prefix(github.com/confluentinc/cli/) --custom-order (gci)` would you have any ideas on what will cause this?

File: test/flink_test.go [hunk 4]
```diff
@@ -1,5 +1,6 @@
 package test
 
+
```
Comment (brianstrauch) [inline test/flink_test.go:3]:
```suggestion

```

**Comment by brianstrauch** [comment]
> @brianstrauch looked into ci failure. `test/flink_test.go:3: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/confluentinc/) -s prefix(github.com/confluentinc/cli/) --custom-order (gci)` would you have any ideas on what will cause this?

I think my latest suggestion will fix it. Not sure why that line was added...

File: test/flink_test.go (hunk 3 above)
Comment (brianstrauch) [reply test/flink_test.go:38]: "Yes" you agree, or "yes" we need it to be 10KB?

**Comment by NicoleYWDu** [comment]
> > @brianstrauch looked into ci failure. `test/flink_test.go:3: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/confluentinc/) -s prefix(github.com/confluentinc/cli/) --custom-order (gci)` would you have any ideas on what will cause this?
> 
> I think my latest suggestion will fix it. Not sure why that line was added...

Seems still has the similar issue:(

**Comment by brianstrauch** [comment]
> > > @brianstrauch looked into ci failure. `test/flink_test.go:3: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/confluentinc/) -s prefix(github.com/confluentinc/cli/) --custom-order (gci)` would you have any ideas on what will cause this?
> > 
> > 
> > I think my latest suggestion will fix it. Not sure why that line was added...
> 
> Seems still has the similar issue:(

You deleted both lines instead of just one :)

**Review by brianstrauch — APPROVED** [review]

**Comment by brianstrauch** [comment]
Please respond to https://github.com/confluentinc/cli/pull/2687#discussion_r1526538518 before merging!

File: test/flink_test.go (hunk 3 above)
Comment (NicoleYWDu) [reply test/flink_test.go:37]: This file is ok for testing. This jar file is simple and for helloworld function. As long as the file size not larger than 1G, we can create the artifact.

**Pull request merged**